libsignal-service/src/sealed_session_cipher.rs

```diff
@@ -43,6 +46,12 @@ pub enum SealedSessionError {
 
     #[error("recipient not trusted")]
     NoSessionWithRecipient,
+
+    #[error("Supplied phone number could not be parsed")]
+    InvalidE164Error(#[from] uuid::Error),
```

Comment:
Maybe `InvalidPhoneNumber` and `"Supplied phone number could not be parsed in E164 format"`?